### PR TITLE
update locator for UserDateDetail

### PR DESCRIPTION
**Pull request by akus062381**
Tiago and I worked on this so that we would have better locators for dates shown on the Detail screens.

### CODE REVIEW

File: awx/ui/src/components/DetailList/UserDateDetail.js [hunk 1]
```diff
@@ -11,13 +11,13 @@ const Detail = styled(_Detail)`
...
 
-function UserDateDetail({ label, date, user, dataCy = null }) {
+function UserDateDetail({ label, date, user }) {
   const dateStr = formatDateString(date);
   const username = user ? user.username : '';
   return (
     <Detail
       label={label}
-      dataCy={dataCy}
+      dataCy="date-detail"
```
Comment (nixocio) [inline awx/ui/src/components/DetailList/UserDateDetail.js]:
```suggestion
      dataCy="user-date-detail"
```

Comment (nixocio) [reply awx/ui/src/components/DetailList/UserDateDetail.js]: This change will possibly generate duplicate `data-cy` on the same page. If that is ok.

**Review by tiagodread — APPROVED** [review]
Looks good to me, if you end up updating the locators to `user-date-detail`don't forget to update the tests too.

**Comment by akus062381** [comment]
closing in favor of https://github.com/ansible/awx/pull/11180

**Pull request closed without merging**